x-pack/plugins/synthetics/server/legacy_uptime/lib/saved_objects/synthetics_monitor.ts

```diff
@@ -109,6 +109,9 @@ export const getSyntheticsMonitorSavedObjectType = (
             },
           },
         },
+        enabled: {
```

Comment:
It's a little hard to follow the types for the saved object so I'll just ask here: Since you're adding a field to the mappings, is that already included in the scope of the 8.6 migration? There aren't any changes to that migration definition, so I'm assuming it's already covered by spreading the `doc.attributes`.